### ISSUE TITLE
winget link

### ISSUE THREAD

**Issue by wincmd64** [erickutcher/httpdownloader#291]
Hi
I can't find 'httpdownloader' via winget.

**Comment by erickutcher** [comment]
There's really no need for a package manager. Just download the zip, extract it to wherever you want, and then run it. https://erickutcher.github.io/#HTTP_Downloader

**Comment by wincmd64** [comment]
...and when you release a new version - again download and again unpack. With winget I update all software with one command:
**winget upgrade --all**

**Comment by erickutcher** [comment]
I don't want to manage that stuff myself, but if someone wants to upload it to their repository, then I don't mind. As long as it's unmodified from the releases on GitHub, then it's fine with me.

**Comment by JNavas2** [comment]
@erickutcher 
> I don't want to manage that stuff myself, but if someone wants to upload it to their repository, then I don't mind. As long as it's unmodified from the releases on GitHub, then it's fine with me.

That's unfortunate. Winget is a powerful and widely used tool that greatly facilitates keeping apps updated. It's a real pain to update apps individually. Please reconsider. (Otherwise I may abandon your app.)

**Comment by wincmd64** [comment]
winget install erickutcher.httpdownloader

**Comment by JNavas2** [comment]
> winget install erickutcher.httpdownloader

Badly out of date. 👎

**Comment by JNavas2** [comment]
- [[microsoft/winget-pkgs] [Package Request]: HTTP Downloader (Issue #322018)](https://github.com/microsoft/winget-pkgs/issues/322018)

**Comment by erickutcher** [comment]
> * [[microsoft/winget-pkgs] [Package Request]: HTTP Downloader (Issue #322018)](https://github.com/microsoft/winget-pkgs/issues/322018)

It looks like it was updated for it.

Like I said, I don't mind if someone else wants to maintain those repositories, but I've given up dealing with anything beyond this site and the extension stores. Too many hoops.

**Comment by JNavas2** [comment]
No, the current package is out of date, but a pull request has been opened.
Like I said, that's unfortunate, because it's fast and easy to create a
pull request.
